plugins/parsers/json_v2/parser_test.go

```diff
@@ -100,6 +100,16 @@ func TestMultipleConfigs(t *testing.T) {
 	}
 }
 
+func TestParserEmptyConfig(t *testing.T) {
+	plugin := &json_v2.Parser{
+		Configs: []json_v2.Config{},
+	}
+
+	err := plugin.Init()
+	require.Error(t, err)
+	require.Equal(t, "no configuration provided", err.Error())
```

Comment:
This can be a one-liner
```suggestion
	require.ErrorContains(t, plugin.Init(), "no configuration provided")
```